apps/src/code-studio/header.js

```diff
@@ -146,6 +157,12 @@ header.build = function (scriptData, stageData, progressData, currentLevelId, pu
   }
 };
 
+const ConnectedShareDialog = connect(state => ({
+  isOpen: state.header.shareDialog.isOpen,
+}), dispatch => ({
+  onClose: () => dispatch(hideShareDialog())
+}))(ShareDialog);
```

Comment:
I would like to better understand why this is the case. This PR seems complex enough that it might be worth talking through in person at some point.